fedimint-server/src/net/p2p_connector.rs

```diff
@@ -58,26 +50,55 @@ pub struct TlsConfig {
 }
 
 /// TCP connector with encryption and authentication
-#[derive(Debug)]
 pub struct TlsTcpConnector {
```

Comment:
Honestly, between "connector", "listener" and "acceptor" I'm getting a little bit confused, as they all means somewhat the same thing. :D